[debug] PhaseRunner.scala:78 Success [id => rbush, phase => typescript] 
[info ] Phase1ReadTypescript.scala:152 Preprocessing TsHelperFile(InFile(/home/tso/tmp/tso-cache/DefinitelyTyped/types/rbush/index.d.ts),FromFolder(InFolder(/home/tso/tmp/tso-cache/DefinitelyTyped/types/rbush),TsIdentLibrarySimple(rbush)),TsIdentModule(None,List(rbush))) [id => rbush, phase => typescript] 
[warn ] Phase1ReadTypescript.scala:190 Processing TsIdentLibrarySimple(rbush) [id => rbush, phase => typescript] 
[debug] PhaseRunner.scala:78 Success [id => rbush, phase => typescript] 
[warn ] Phase2ToScalaJs.scala:34 Processing TsIdentLibrarySimple(rbush) [id => rbush, phase => scala.js] 
[debug] PhaseRunner.scala:78 Success [id => rbush, phase => scala.js] 
[warn ] Phase3CompileBloop.scala:211 Built rbush in 821 ms [id => rbush, phase => build] 
[info ] Phase3CompileBloop.scala:221 published local, [/home/tso/.ivy2/local/com.scalablytyped/rbush_sjs0.6_2.12/2.0.1-dt-20180910Z-27fac7/jars/rbush_sjs0.6_2.12.jar, /home/tso/.ivy2/local/com.scalablytyped/rbush_sjs0.6_2.12/2.0.1-dt-20180910Z-27fac7/srcs/rbush_sjs0.6_2.12-sources.jar, /home/tso/.ivy2/local/com.scalablytyped/rbush_sjs0.6_2.12/2.0.1-dt-20180910Z-27fac7/ivys/ivy.xml, /home/tso/.ivy2/local/com.scalablytyped/rbush_sjs0.6_2.12/2.0.1-dt-20180910Z-27fac7/poms/rbush_sjs0.6_2.12.pom] [id => rbush, phase => build] 
[debug] PhaseRunner.scala:78 Success [id => rbush, phase => build] 
[error] PhaseRes.scala:75 Caught exception [id => rbush, phase => publish] java.util.concurrent.ExecutionException: java.net.ConnectException: connection timed out: api.bintray.com/108.168.194.93:443
	at com.ning.http.client.providers.netty.future.NettyResponseFuture.abort(NettyResponseFuture.java:229)
	at com.ning.http.client.providers.netty.request.NettyConnectListener.onFutureFailure(NettyConnectListener.java:107)
	at com.ning.http.client.providers.netty.request.NettyConnectListener.operationComplete(NettyConnectListener.java:115)
	at org.jboss.netty.channel.DefaultChannelFuture.notifyListener(DefaultChannelFuture.java:409)
	at org.jboss.netty.channel.DefaultChannelFuture.notifyListeners(DefaultChannelFuture.java:400)
	at org.jboss.netty.channel.DefaultChannelFuture.setFailure(DefaultChannelFuture.java:362)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:142)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:337)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: connection timed out: api.bintray.com/108.168.194.93:443
	at com.ning.http.client.providers.netty.request.NettyConnectListener.onFutureFailure(NettyConnectListener.java:103)
	... 13 more
Caused by: org.jboss.netty.channel.ConnectTimeoutException: connection timed out: api.bintray.com/108.168.194.93:443
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:139)
	... 8 more

[debug] PhaseRunner.scala:82 Failure, [rbush => java.util.concurrent.ExecutionException: java.net.ConnectException: connection timed out: api.bintray.com/108.168.194.93:443] [id => rbush, phase => publish] 
